lib/ModuleInfo.js

```diff
@@ -81,12 +81,40 @@ const ModuleInfoPlatform = {
 
 const HEADER_SIZE = 24;
 
+function moduleFunctionToChar(f) {
+    let moduleFunction;
+
+    if (f && f < ModuleFunctionToChar.length) {
+        moduleFunction = ModuleFunctionToChar[f];
+    }
+
+    return moduleFunction;
+}
+
+function addDependency(to, f, index, version) {
+    const depFunction = moduleFunctionToChar(f);
+    if (depFunction) {
+        to.push({f: depFunction, v: version, n: ''+index});
+    }
+}
+
+
+function parsedModuleToDescribe(info) {
+    const prefix = info.prefixInfo;
+    const moduleFunction = moduleFunctionToChar(prefix.moduleFunction);
+    const dependencies = [];
+    addDependency(dependencies, prefix.depModuleFunction, prefix.depModuleIndex, prefix.depModuleVersion);
+    addDependency(dependencies, prefix.dep2ModuleFunction, prefix.dep2ModuleIndex, prefix.dep2ModuleVersion);
+    return { f: moduleFunction, n: ''+prefix.moduleIndex, v: prefix.moduleVersion, d: dependencies };
+}
+
 module.exports = {
     FunctionType: ModuleFunction,
     FunctionChar: ModuleFunctionChar,
     Flags: ModuleInfoFlags,
     FunctionToChar: ModuleFunctionToChar,
     ValidationFlags: ModuleValidationFlags,
     Platform: ModuleInfoPlatform,
-    HEADER_SIZE: HEADER_SIZE
+    HEADER_SIZE: HEADER_SIZE,
+    parsedModuleToDescribe,
```

Comment:
`parsedModuleToDescribe: parsedModuleToDescribe` ?

Reply:
We can safely target ES6 now, but I agree this should probably be done in a major release.